tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -830,6 +842,7 @@ static PyObject * TypeError_to_NotImplemented_(PyObject* self, PyObject* args, P
 // Any new ops added here should be accompanied with a commment why they are not
 // being registered through native_functions.yaml, and include reviwers from C++/JIT
 PyMethodDef variable_methods[] = {
+  // magic method bound here to wrap NotImplementedError
```

Comment:
nit: "These magic methods are all bound here to wrap NotImplementedError" (some sort of plural statement"